### PR TITLE
feat(trivy): get trivy-fs working

### PR DESCRIPTION
**Pull request by sxlijin**

### CODE REVIEW

**Comment by sxlijin** [comment]
Waiting for https://github.com/trunk-io/trunk/actions/runs/4139449682 to finish.

File: .trunk/trunk.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: 0.1
 
 cli:
-  version: 1.4.2-beta.6
+  version: 1.4.2-beta.8
```
Comment (TylerJang27) [inline .trunk/trunk.yaml:4]: When this lands I'll go in and fix renovate and remark-lint

File: linters/trivy/plugin.yaml [hunk 2]
```diff
@@ -16,13 +16,21 @@ lint:
 
   definitions:
     - name: trivy
+      files: [lockfile]
       download: trivy
       known_good_version: 0.37.1
+      # Trivy does not support batching
+      batch: false
```
Comment (TylerJang27) [inline linters/trivy/plugin.yaml:23]: This might still work, but `batch` is better as a commands: field.

**Review by TylerJang27 — APPROVED** [review]
lgtm, aside from maybe the batch nit

**Pull request merged**